trapper/pipelines/question_answering_pipeline.py

```diff
@@ -68,17 +68,21 @@ def normalize(self, item):
                 elif isinstance(item[k], str) and len(item[k]) == 0:
                     raise ValueError("`{}` cannot be empty".format(k))
 
-            question = {"text": item["question"], "start": None}
-            item["question"] = self._convert_to_span_tuple(question)
+            self._add_id(item)
             return item
         raise ValueError("{} argument needs to be of type dict".format(item))
 
     @staticmethod
     def _convert_to_span_tuple(span: Union[SpanDict, SpanTuple]) -> SpanTuple:
-        if isinstance(span, SpanDict):
+        if isinstance(span, dict):
             span = convert_spandict_to_spantuple(span)
         return span
 
+    @staticmethod
+    def _add_id(item: Dict) -> None:
```

Comment:
why do we do that? "id" is fine as a key.

Reply:
we pass as `(**example)`, I did not want to change that, so i manipulate `id` as `id_` for argument name `id_`.